docs/apereo-incubation.md

```diff
@@ -40,8 +42,7 @@ Most major contributors have ICLAs on file.
 
 Next actions:
 
-+ Audit for CLA completeness of contributors
-+ Document CLA requirements in `CONTRIBUTING.md`
++ Follow up with past Contributors to secure CLAs
```

Comment:
> Cf. thoughts about partially adopting cla-assistant.io. On skim looks like CLAHub and cla-assistant.io are similar

Which tool is less important than automating the process.

>  [concern that click-through ICLA agreement is not sufficient](https://groups.google.com/a/apereo.org/d/msg/projects/elNXbx5F-dE/Tu2xEqYhAgAJ)

I'm getting "You must be a member of this group to view and participate in it. Membership is by invitation only.", but I trust that there are exciting things being discussed.